### DISCUSSION TITLE
Pagination and Filters for Presets & Bookmarks in sidebar

### DISCUSSION THREAD

**Discussion by licitdev** [directus/directus#6477] [Feature Requests]
Currently, the sidebar for Presets & Bookmarks only contains an information tab. As the number of users increases, the automatically created presets may be hard to manage. Hence, pagination and filters could come in handy similar to how we manage the collections.

Current:
<img width="297" alt="Screenshot 2021-06-24 at 6 36 33 PM" src="https://user-images.githubusercontent.com/26413686/123249467-8eb51f00-d51b-11eb-8a88-2a996ded6da1.png">

Ideal with Filter:
<img width="297" alt="Screenshot 2021-06-24 at 6 36 16 PM" src="https://user-images.githubusercontent.com/26413686/123249554-a9879380-d51b-11eb-8500-6dd20dd99c34.png">

**Comment by azrikahar** [comment]
Implemented by #10978 in `9.8.0`!
